api/src/plugins/not-found.ts

```diff
@@ -15,8 +15,11 @@ const fourOhFour: FastifyPluginCallback = (fastify, _options, done) => {
   // If the request accepts JSON and does not specifically prefer text/html,
   // this will return a 404 JSON response. Everything else will be redirected.
   fastify.setNotFoundHandler((request, reply) => {
+    const logger = fastify.log.child({ request });
     const accepted = request.accepts().type(['json', 'html']);
```

Comment:
I forgot: The serializer only handles it if the object contains exactly `{ req }`

```suggestion
  fastify.setNotFoundHandler((req, reply) => {
    const logger = fastify.log.child({ req });
    const accepted = req.accepts().type(['json', 'html']);
```

Reply:
Remember the line I cannot edit at the end of the callback.